pynucastro/networks/rate_collection.py

```diff
@@ -765,8 +765,10 @@ def _constraint_eq(self, u, rho, T, ye):
 
         comp_NSE = self._evaluate_comp_NSE(u, rho, T, ye)
 
-        eq1 = sum(comp_NSE.X.values()) - 1.0
-        eq2 = ye - comp_NSE.eval_ye()
+        nse_ye = sum(nuc.Z * comp_NSE.X[nuc] / nuc.A for nuc in self.unique_nuclei)
```

Comment:
can you add a comment here explaining why we don't use `eval_ye()`

Reply:
Okay! Done!